pandas/tests/reshape/test_reshape.py

```diff
@@ -645,3 +645,21 @@ def test_reshaping_multi_index_categorical(self):
             index=dti.rename("major"),
         )
         tm.assert_frame_equal(result, expected)
+
+class TestMultiIndexReshape:
+    def test_unstacking_multi_index_df(self):
```

Comment:
can you move this test to pandas/tests/frame/test_reshape.py

Reply:
Sure will do that.

Reply:
Done